rust/crd/src/lib.rs

```diff
@@ -283,20 +292,20 @@ impl Configuration for MetaStoreConfig {
 
     fn compute_cli(
         &self,
-        _resource: &Self::Configurable,
+        resource: &Self::Configurable,
```

Comment:
Nit: please rename to `hive`
Same goes for `compute_env` and `compute_files`

Reply:
done